test/validateConfig.spec.js

```diff
@@ -0,0 +1,47 @@
+/* eslint no-console: 0 */
```

Comment:
we turn off this rule a lot, since this project is a CLI app I think it's better to set this directly in our ESLint config.

thoughts?

Reply:
Agreed.